app/script/view_model/content/MessageListViewModel.js

```diff
@@ -216,6 +218,13 @@ z.viewModel.content.MessageListViewModel = class MessageListViewModel {
       });
   }
 
+  _getMessagesContainer() {
```

Comment:
This is bonus. As the messages list element doesn't change during the lifecycle of a conversation, we can safely cache it.